utils/trajectory/TrajectoryData.js

```diff
@@ -0,0 +1,30 @@
+const fsp = require("fs").promises;
```

Comment:
Avoid having a duplicate `TrajectoryData.js` (there is also one in `test/neural/TrajectoryData.js`)